app/models/notification.rb

```diff
@@ -36,7 +36,7 @@ def parent_event
   end
 
   def should_deliver?
-    enabled? && subscribed? && permitted?
+    not_hidden_if_onprem_multitenancy? && subscribed? && permitted?
```

Comment:
oh god... it's hard to understand the meaning of this :upside_down_face:

Reply:
yeah, I don't know what `not_hidden_if_onprem_multitenancy?` means but the behavior is the same as previously with the notifications feature enabled.